config/backend-config/types.go

```diff
@@ -87,13 +87,29 @@ type ConfigT struct {
 	Sources         []SourceT       `json:"sources"`
 	Libraries       LibrariesT      `json:"libraries"`
 	ConnectionFlags ConnectionFlags `json:"flags"`
+	Settings        Settings        `json:"settings"`
 }
 
 type ConnectionFlags struct {
 	URL      string          `json:"url"`
 	Services map[string]bool `json:"services"`
 }
 
+type Settings struct {
+	DataRetention DataRetention `json:"dataRetention"`
+}
+
+type DataRetention struct {
+	EnableReportingPii     bool          `json:"enableReportingPii"`
+	UseRudderServerStorage bool          `json:"useRudderServerStorage"`
```

Comment:
If the default value for both is `false` we are fine. But I think that the default value for UseRudderServerStorage is true, correct? Maybe it would be wiser to reverse this config option and change its name accordingly